server/channels/app/users/password_test.go

```diff
@@ -13,13 +13,6 @@ import (
 	"github.com/mattermost/mattermost/server/public/model"
 )
 
-func TestComparePassword(t *testing.T) {
-	hash := HashPassword("Test")
```

Comment:
How come you're getting rid of this?

Reply:
@enzowritescode That `HashPassword` was never used anywhere, so removing the dead code.